src/jira.py

```diff
@@ -2,17 +2,19 @@
 import re
 import requests
 
-from settings import default_orgs, transitions
+from more_itertools import first
+from settings import transitions
 from src.access import get_access_params
 from src.issue import Issue
+from src.utilities import get_zenhub_pipeline
 
 
 class JiraBoard:
 
-    def __init__(self, project_key: str):
+    def __init__(self, project_key: str, org: str):
         """Create a Project storing all issues belonging to the provided project key"""
 
-        self.url = get_access_params('jira')['options']['server'] + default_orgs['jira']
+        self.url = get_access_params('jira')['options']['server'] % org
```

Comment:
I don't think this string substitution will work. Please correct. (Not sure if this class is used currently, but it looks like we need it later).

Reply:
I take the above back, it's actually called a string interpolation, and **it works**. I'm not sure if this is a good solution to what we want to achieve here, it seems a bit convoluted. It took me a while to actually find the `%s`, particularly as it's not in def `get_access_params`, but in settings.
At the very least I would correct in `settings.py` the comment following that string containing the interpolation. It incorrectly says that _repo_ will be interpolated when it is actually the _organization_.

Reply:
Fixed the comment. I agree it's kind of confusing, not sure how else to accomplish this though, since the org name needs to go in the middle of the url